exercise_data/data_explanations.md

```diff
@@ -14,4 +14,5 @@
 14 = Dataset 93 of https://webdav.tuebingen.mpg.de/cause-effect/ (citation = J. M. Mooij, J. Peters, D. Janzing, J. Zscheischler, B. Schoelkopf: "Distinguishing cause from effect using observational data: methods and benchmarks"). First column of data is average precipitation over 1948 to 2004 in mm/day, second is average runoff
 15 = Same as 14, but dataset 73: first column is CO2 emissions, second column is energy use (kg of oil equivalent)
 16 = Long-time simulation data for orbital parameters of Earth, from http://vo.imcce.fr/insola/earth/online/earth/online/index.php with DOI: 10.1051/0004-6361:20041335. The data (each column) are: time (1000yr), eccentricity, climatic precession, obliquity (radians), insolation (W/m^2). Latitude is 65, record at day of summer solstice (21 June, true longitude = 90).
-17 = Vostok Ice Core Data for 420,000 Years. First column is time in years (increasing towards the past), second column is estimated temperature difference w.r.t. current day temperature. Notice that the data are not equi-temporally sampled. The data were downloaded from https://www.ncdc.noaa.gov/paleo-search/ with suggested data citation: "Petit, J.R., et al., 2001, Vostok Ice Core Data for 420,000 Years, IGBP PAGES/World Data Center for Paleoclimatology Data Contribution Series #2001-076. NOAA/NGDC Paleoclimatology Program, Boulder CO, USA." and see https://doi.org/10.1038/20859 
+17 = Vostok Ice Core Data for 420,000 Years. First column is time in years (increasing towards the past), second column is estimated temperature difference w.r.t. current day temperature. Notice that the data are not equi-temporally sampled. The data were downloaded from https://www.ncdc.noaa.gov/paleo-search/ with suggested data citation: "Petit, J.R., et al., 2001, Vostok Ice Core Data for 420,000 Years, IGBP PAGES/World Data Center for Paleoclimatology Data Contribution Series #2001-076. NOAA/NGDC Paleoclimatology Program, Boulder CO, USA." and see https://doi.org/10.1038/20859
+18 = EEG recordings of a patient from the Siena Scalp EEG Database. Each Column is one EEG signal at electrode F8 (right frontal region, patient has right-hemisphere seizures) from patient 0, downsampled to 64Hz. The seizures in the recordings are documented to have started at t=[1143.0s, 1220.0s, 1006.0s, 904.0s] and ended at t=[1213.0s, 1274.0s, 1080.0s, 971.0s], respectively. These time series were downloaded from https://physionet.org/files/siena-scalp-eeg/1.0.0/PN00/PN00-1.edf and converted into the csv format using convert_18.py
```

Comment:
Are these times that you document here the times of the real data, of the subsampled data? Isn't it simpler to give row indices as the start and end?